CONTRIBUTING.md

```diff
@@ -1,86 +1,187 @@
+-   If you find an issue you would like to fix, [open a pull request](#pull-requests). Issues tagged as [_Good first issue_](https://github.com/pact-foundation/pact-python/labels/Good%20first%20issue) are a good place to get started.
+-   Read through the [docs](https://docs.pact.io). If you find anything that is confusing or can be improved, you can click "Edit this page" at the bottom of most docs, which takes you to the GitHub interface to make and propose changes.
+-   Take a look at the [features requested](https://github.com/pact-foundation/pact-python/labels/feature) by others in the community and consider opening a pull request if you see something you want to work on.
 
-We also appreciate it if you take the time to update and write tests for any changes
-you submit.
+Contributions are very welcome. If you think you need help planning your contribution, please ping us on [Slack](https://slack.pact.io) and let us know you are looking for a bit of help.
 
-[examples]: https://github.com/pact-foundation/pact-python/tree/master/examples
+### Join our Slack
 
-## Commit messages
+We have a [Slack](https://slack.pact.io) to discuss all things about Pact and its development. Feel free to ask questions about Pact Python specifically in the [`#pact-python`](https://pact-foundation.slack.com/archives/C9VECUP6E) channel, or broader questions about the Pact ecosystem over in the [`#general`](https://pact-foundation.slack.com/archives/C5F4KFKR8) channel.
 
-pact-python is adopting the [Conventional Commits](https://www.conventionalcommits.org)
-convention. Please ensure you follow the guidelines, we don't want to be that person,
-but the commit messages are very important to the automation of our release process.
+### Triaging Issues and Pull Requests
 
-Take a look at the git history (`git log`) to get the gist of it.
+One great way you can contribute to the project without writing any code is to help triage issues and pull requests as they come in.
 
-If you'd like to get some CLI assistance there is a node npm package. Example usage is:
+-   Ask for more information if you believe the issue does not provide all the details required to solve it.
+-   Suggest [labels](https://github.com/pact-foundation/pact-python/labels) that can help categorize issues.
+-   Flag issues that are stale or that should be closed.
+-   Ask for test plans and review code.
 
-```shell
-npm install -g commitizen
-npm install -g cz-conventional-changelog
-echo '{ "path": "cz-conventional-changelog" }' > ~/.czrc
-```
+## Our Development Process
 
-When you commit with Commitizen, you'll be prompted to fill out any required
-commit fields at commit time. Simply use `git cz` or just `cz` instead of
-`git commit` when committing. You can also use `git-cz`, which is an alias
-for `cz`.
+Pact Python uses [GitHub](https://github.com/pact-foundation/pact-python) as its source of truth. The team will be working directly there. All changes will be public from the beginning.
```

Comment:
how do we balance GitHub being a source of truth and telling people to start ideas off in Slack, how do we trace back conversations to GitHub.

Links to slack aren't great, as content gets lost after 90 days, unless we then link to the archives.

Reply:
The way I see it (and let me know if other people are of the same opinion) is that Slack is useful for more informal discussions and quick Q&As. If a good idea comes out of Slack, then it should be recorded in a GitHub Issue or Discussion for a more permanent record. In this way, Slack does not really serve a role as a source of truth.

Reply:
totally makes sense to me, thanks for being explicit here for users